core/src/main/java/gyro/core/command/StateCopyCommand.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.core.command;
+
+import java.nio.file.Path;
+
+import gyro.core.FileBackend;
+import gyro.core.GyroCore;
+import gyro.core.GyroException;
+import gyro.core.LocalFileBackend;
+import gyro.core.RemoteStateBackend;
+import io.airlift.airline.Command;
+import io.airlift.airline.Help;
+import io.airlift.airline.Option;
+import io.airlift.airline.model.MetadataLoader;
+
+@Command(name = "copy", description = "Copy state files between local and remote backend.")
+public class StateCopyCommand implements GyroCommand {
+
+    @Option(name = "--to-local", description = "Pull state files from a remote backend and copy to local backend")
```

Comment:
Given the other suggestion to allow named state backends this should be changed to so that `--to` and `--from` take names and copies between those names.

In this case the name `local` would be a special name referring to the `LocalFileBackend`.